checker/exported_ident_doc_checker.go

```diff
@@ -3,19 +3,39 @@ package checker
 import (
 	"fmt"
 	"go/ast"
+	"strings"
+
+	"github.com/uber-go/mapdecode"
 )
 
 func init() {
 	must(Register("exported_ident_doc", NewExportedIdentDocChecker))
 }
 
+// ExportedIdentDocCheckerConfig describes the configuration of a ExportedIdentDocChecker.
+type ExportedIdentDocCheckerConfig struct {
+
+	// CheckDocPrefix signals if the checker should ensure that every doc comment begins
+	// with the name of the item it describes.
+	CheckDocPrefix bool `mapdecode:"check_doc_prefix"`
```

Comment:
How about changing the flag name to `has_ident_prefix`? I'm thinking that since the rule is named `exported_ident_doc` it is already clear that we're in the context of ident doc, the only thing that we have to specify is that we want the line to have the ident as a prefix.

Reply:
Good point! Changed it.